man/repart.d.xml

```diff
@@ -734,6 +734,33 @@
 
         <xi:include href="version-info.xml" xpointer="v253"/></listitem>
       </varlistentry>
+
+      <varlistentry>
+        <term><varname>FSTab=</varname></term>
```

Comment:
I would rename this to `Mountpoint=` (or `MountPoint=`?).

i.e. focus on what you configure here, not on how you output it.

(i.e. we want to keep the door open to generate other formats from this, for example native systemd .mount units)